### PR TITLE
feat(FWS-32): Ability To Get Fitness Package By Id

### PR DESCRIPTION
**Pull request by AlexeiTimbro**
**JIRA: link to jira ticket**
https://champlainsaintlambert.atlassian.net/browse/FWS-32

**What is the ticket about and why are we making this change?**

This ticket is about making a details button for each fitness package to add more information shown for each fitness package that is important for clients when they choose the type of fitness package that they want to choose. It also allows them to book from the pop window with the book button added to the pop-up window.

**What are the various changes and what other modules do those changes affect?**
 
-4 types of testing
-backend for the get by Id
-Front end for the get by Id
- Pop-up window
- Made components for all the nav and footers
- Changed the nav and footer styling

**UI Change:**

Before:
image

After: 
image

After clicking the Details button: 
image

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\fitwsarah_frontend",
+    "\\fitwsarah_frontend\\public",
+    "\\fitwsarah_frontend\\src"
+  ],
+  "PreviewInSolutionExplorer": false
+}
```
Comment (emilegirardGit) [inline .vs/VSWorkspaceState.json:9]: Can you explain what this does?

File: fitwsarah_frontend/src/components/footer/footerLoggedIn/footerLoggedIn.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import './footerLoggedIn.css'; 
+
+function FooterLoggedIn() {
+  return (
+    <div className="footer-logged-in">
+      {/* Content goes here */}
+    </div>
+  );
+}
+export default FooterLoggedIn;
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/components/footer/footerLoggedIn/footerLoggedIn.js:11]: Did you forget to add the footer?

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (AlexeiTimbro) [reply .vs/VSWorkspaceState.json:9]: This code from a VSWorkspaceState.json file specifies which folders are expanded in Visual Studio's Solution Explorer and indicates that the preview feature for selected items is disabled.

File: fitwsarah_frontend/src/components/footer/footerLoggedIn/footerLoggedIn.js (hunk 2 above)
Comment (AlexeiTimbro) [reply fitwsarah_frontend/src/components/footer/footerLoggedIn/footerLoggedIn.js:11]: No, I simply haven't made the logged-in footer so far I just added it in the files to make sure that when we actually make it we don't waste time configuring it..

**Review by emilegirardGit — APPROVED** [review]
Good work

**Pull request merged**